src/styles/settings.plugin.scss

```diff
@@ -110,3 +113,17 @@
 .jw-settings-submenu-button {
 	display: none !important;
 }
+
+// caption styling
+// @see https://developer.mozilla.org/en-US/docs/Web/CSS/::cue
+::cue {
```

Comment:
It will style any caption on site, can we limit the scope?

Reply:
I tried but the browsers seem to ignore that :/

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=865395
Did you check how it works in FF?

Reply:
@hakubo, @bkoval , i've removed `::cue` completely in favor of => https://github.com/Wikia/jwplayer-fandom/pull/19/files#diff-328f91fce83d078dfc42789912f79cbbR103